### DISCUSSION TITLE
USerWarning: No benchmark

### DISCUSSION THREAD

**Discussion by MagicHealer** [ContinualAI/avalanche#1266] [Q&A]
UserWarning: No benchmark provided to the evaluation plugin. Metrics may be computed on inconsistent portion of streams, use at your own risk.
  warnings.warn(

How can I deal with this?

**Comment by AntonioCarta** [comment]
Hi @MagicHealer. Are you on the last version (0.3.1)? the benchmark argument is not needed anymore but we may have forgotten to remove some warnings.
